tracing-attributes/src/lib.rs

```diff
@@ -227,10 +227,20 @@ mod expand;
 ///     42
 /// }
 /// ```
-/// The return value event will have the same level as the span generated by `#[instrument]`.
-/// By default, this will be `TRACE`, but if the level is overridden, the event will be at the same
+/// The return value event will by default have the same level as the span generated by `#[instrument]`.
+/// By default, this will be `TRACE`, but if the span level is overridden, the event will be at the same
```

Comment:
tiny nit: two sentences that both say "by default" feels _slightly_ awkward, what do you think about:
```suggestion
/// The level of the return value event defaults to the same level as the span generated by `#[instrument]`.
/// By default, this will be `TRACE`, but if the span level is overridden, the event will be at the same
```
or something?